### PR TITLE
List highest versions per repository

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/7428
closes #7428

### CODE REVIEW

**Comment by bmbouter** [comment]
@fao89 is there a low-effort test that could come with this?

**Comment by fao89** [comment]
> @fao89 is there a low-effort test that could come with this?

mid-effort, I can do it

**Review by bmbouter — APPROVED** [review]
This looks great to me. Thank you for the fix and the test too! :rocket:

File: pulp_ansible/app/galaxy/v3/views.py [hunk 1]
```diff
@@ -113,10 +113,17 @@ def get_queryset(self):
...
-            pk__in=distro_content, is_highest=True
+        versions = CollectionVersion.objects.filter(pk__in=distro_content).values_list(
+            "version", flat=True
         )
+
+        if len(versions):
+            highest = sorted(versions, key=semantic_version.Version, reverse=True)[0]
+            query_params["version"] = highest
+
+        collections = CollectionVersion.objects.select_related("collection").filter(**query_params)
```
Comment (fao89) [inline pulp_ansible/app/galaxy/v3/views.py:126]: I found a bug here

**Pull request merged**